styles.css

```diff
@@ -1,151 +1,108 @@
-
-
-.container
-{
-    display: flex;
-    flex-direction: column;
-   
+.container {
+  display: flex;
+  flex-direction: column;
 }
-.box1{
-  /* My Logo */
-
-
-position: static;
-width: 76px;
-height: 20px;
-left: 12px;
-top: 10px;
-
-font-family: Poppins;
-font-style: normal;
-font-weight: bold;
-font-size: 18px;
-line-height: 20px;
-/* identical to box height, or 111% */
```

Comment:
- Take your project to the next level. Kindly, remove unused commented lines and delete blank spaces, that will make your code more readable.